### PR TITLE
genymotion-cloud-saas-start 0.0.1

### PR DESCRIPTION
**Pull request by thomascarpentier**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/genymotion-cloud-saas-start/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+- ubuntu-16.04
+project_type_tags:
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- utility
```
Comment (koral--) [inline steps/genymotion-cloud-saas-start/0.0.1/step.yml]: It should be rather `test`

Comment (thomascarpentier) [reply steps/genymotion-cloud-saas-start/0.0.1/step.yml]: @godrei said me to put utility as it does not perform any test by itself

Comment (godrei) [reply steps/genymotion-cloud-saas-start/0.0.1/step.yml]: @koral-- I still keep it, it does not perform/execute any tests.

Comment (koral--) [reply steps/genymotion-cloud-saas-start/0.0.1/step.yml]: There was a similar case and discussion with @godrei in 2017 and the conclusion was opposite.
https://github.com/bitrise-io/bitrise-steplib/pull/1101#issuecomment-314151310

Comment (godrei) [reply steps/genymotion-cloud-saas-start/0.0.1/step.yml]: I see, at that time i stated the same as this case and i still keep it.
The source of this issue is that we did not have a proper guide which describes the project type tags.
As i mentioned previously the new Step Development Guideline is in progress, the Tech Writer team is working on the final touches.

Comment (koral--) [reply steps/genymotion-cloud-saas-start/0.0.1/step.yml]: OK, thx.
If it is ready don't forget to check if existing steps are following new guidelines ;)

**Pull request merged**